### PR TITLE
Feat/index scoped tiered access

### PR DESCRIPTION
**Pull request by ZakirG**
Jira Ticket: https://ctds-planx.atlassian.net/browse/HP-31

### New Features
- Adds support for index-scoped tiered access. New optional config property, tierAccessLevel, in each tab of the explorerConfig block in the portal config. Index-scoped tiered-access only available since Guppy server version 0.11.0. See README for more information. 

### Deployment Changes
- New optional config property, tierAccessLevel, in each tab of the explorerConfig block in the portal config. Used to enable index-scoped tiered-access, which is only available starting with Guppy server version 0.11.0. See README for more information.

### CODE REVIEW

File: src/localconf.js
Comment (mfshao) [inline src/localconf.js:165]: minor: probably should just freak out if there is no `guppyConfig`, that block is required for each explorer tab

**Review by mfshao — COMMENTED** [review]
I think whatever you put into `Dependency updates` should goes into `Deployment changes`

**Comment by mfshao** [comment]
and also might be good to mention index-scoped access level only available since guppy version XXX

**Comment by ZakirG** [comment]
> and also might be good to mention index-scoped access level only available since guppy version XXX

for sure, i'll add that once Marcelo has QA'd my Guppy PR and it's merged and versioned

**Review by mfshao — APPROVED** [review]
personally lgtm, let's merge after https://github.com/uc-cdis/guppy/pull/104

File: src/localconf.js
Comment (ZakirG) [reply src/localconf.js:165]: the localconf.js file does not throw any errors or intentionally abort builds. i'm gonna keep with that convention

**Pull request merged**
